docker-compose/dse-6.8/docker-compose.yml

```diff
@@ -3,9 +3,10 @@ version: '2'
 services:
   dse-1:
     image: datastax/dse-server:${DSETAG}
+    platform: linux/amd64
     networks:
       - stargate
-    mem_limit: 2G
+    mem_limit: 2560M
```

Comment:
Maybe better idea would be to stick to 2G and decrease the max heap to 1G, currently `MAX_HEAP_SIZE=1536M`

Reply:
I tried this and the DSE containers ended up failing about 13 minutes after I started them. So probably this is cutting it too close.

Reply:
Good that you managed to verify it at any rate. Definitely prefer working over non-working, but also good we know how close it is.